code/espurna/homeassistant.ino

```diff
@@ -135,6 +137,9 @@ void _haSendSwitches() {
             DynamicJsonBuffer jsonBuffer;
             JsonObject& config = jsonBuffer.createObject();
             _haSendSwitch(i, config);
+            config["uniq_id"] = getIdentifier() + "_switch_" + String(i);
```

Comment:
btw, why not use `type` here?

Reply:
no need. switch_**i** is unique already

Reply:
but I have no problem changing it

Reply:
huh, i think it only matters per platform. just tried changing topic from switch to light and it works ok.
i see that you already changed it though :) so let's keep it with `type`